cursorless-talon/src/cheatsheet_html/cheat_sheet.py

```diff
@@ -12,12 +12,29 @@
 from .sections.special_marks import get_special_marks
 
 mod = Module()
+ctx = Context()
+ctx.matches = r"""
+app: vscode
+"""
 
 cheatsheet_out_dir = Path(tempfile.mkdtemp())
 instructions_url = "https://www.cursorless.org/docs/"
 
 
 @mod.action_class
+class Actions:
+    def cursorless_cheat_sheet_show_html():
+        """Show new cursorless html cheat sheet"""
+        app.notify(
+            'Please first focus an app that supports cursorless, eg say "focus code"'
+        )
```

Comment:
This seems like better UX than Talon just refusing to hear them when they say "cursorless cheatsheet" in the wrong place